src/bundler.ts

```diff
@@ -46,6 +47,7 @@ export interface Options {
   // or other hint to fix the top-level folder.
   basePath?: string;
 
+  // TODO(usergenic): Added Imports is not yet supported.
```

Comment:
Hmm.  I say that, but now looking at what vulcanize is actually doing with it per this issue #402 and checking out the test-- The behavior of addedImports seems counter to what one would expect.  The feature was added to vulcanize in #248 but no discussion as to reason why feature was added is given or why it was implemented without inlining.  We should discuss.